mcp-tools/maos-mcp-hub/servers/bitbucket/tools.py

```
@@ -1957,7 +1967,7 @@ async def search_learned_fixes(
 
 
 # @mcp.tool() -- Removed: Hub handles registration
-async def get_knowledge_base_stats() -> dict:
+async def get_knowledge_base_stats(repo_slug: str = "") -> dict:
```

Ruff (0.15.2)

[warning] Unused function argument: `repo_slug`

(ARG001)